torch/export/exported_program.py

```diff
@@ -1208,7 +1210,9 @@ def example_inputs(self, value):
     @property
     @compatibility(is_backward_compatible=False)
     def call_spec(self):
-        CallSpec = namedtuple("CallSpec", ["in_spec", "out_spec"])
+        class CallSpec(NamedTuple):
```

Comment:
Thank you for this! <3 Reallly need to replace the rest of these legacy NamedTuples